General queries/System Guard Security Level Baseline.txt

```diff
@@ -0,0 +1,10 @@
+// Establishes a baseline SystemGuardSecurityLevel and show the devices that are below that baseline
```

Comment:
Any chance you add a comment on what this baseline mean? Maybe point to a doc URL?